sass/directives/_flyout.scss

```diff
@@ -87,3 +87,117 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+// Structure of markup should be:
+@mixin mobile-subnav-flyout() {
+  width: auto; 
+  display: none;
+  position: relative;
+  cursor: pointer;
+
+  @media only screen and (max-width: $small-screen-max) {
+    line-height: 45px;
+    display: block;
+    height: $product_subnav--height;
+    margin-left: -$container-edge-padding;
+    margin-right: -$container-edge-padding;
+  }
+  
+  &:hover {
+    ul {
+      display: block;
+    }
+
+    .mobile-subnav__top-link:after {
+      content: '';
+    }
+  }
+
+  ul {
+    display: none;
+    background-color: white;
+    border-bottom: $base-border;
+    position: absolute !important;
+    top: $product_subnav--height;
+    left: 0;
+    right: 0;
+    width: 100%;
+
+    > li {
+      display: block;
+      line-height: 45px;
+      width: 100%;
+      
+      &:hover {
+        background-color: $base-link-color;
+      }
+
+      a {
+        display: block;
+        float: none;
```

Comment:
No, its really not. And a lot of the code for that navigation can be cleaned up. I did make a note in the other PR (https://github.com/cbdr/employer/pull/890#event-1135093937) about the refactor story. So when it does happen, we can certainly clean this up.